apps/condo/domains/acquiring/hooks/useAcquiringIntegrationContext.ts

```diff
@@ -0,0 +1,56 @@
+import {
+    AcquiringIntegration,
+    AcquiringIntegrationContext, AcquiringIntegrationContextWhereInput,
+    QueryAllAcquiringIntegrationContextsArgs,
+} from '@app/condo/schema'
+import get from 'lodash/get'
+
+import { IRefetchType } from '@open-condo/codegen/generate.hooks'
+import { useOrganization } from '@open-condo/next/organization'
+
+import { AcquiringIntegrationContext as AcquiringIntegrationContextApi, AcquiringIntegration as AcquiringIntegrationApi } from '@condo/domains/acquiring/utils/clientSchema'
+
+type TUseAcquiringIntegrationContext = {
+    loading: boolean,
+    error: string,
+    acquiringIntegration: AcquiringIntegration,
+    acquiringIntegrationContext: AcquiringIntegrationContext,
+    refetchAcquiringIntegrationContext: IRefetchType<AcquiringIntegrationContext, QueryAllAcquiringIntegrationContextsArgs>,
+}
+
+export function useAcquiringIntegrationContext ({ status = undefined, invoiceStatus = undefined } = {}): TUseAcquiringIntegrationContext {
+    const { organization } = useOrganization()
+    const orgId = get(organization, 'id', null)
+
+    // NOTE: On practice there's only 1 acquiring and there's no plans to change it soon
+    const { objs: acquiring, loading: acquiringLoading, error: acquiringError } = AcquiringIntegrationApi.useObjects({
```

Comment:
Strange hook. Why not just fetch context? You can get integration from relation. Instead you're sending x3 requests :)

I think it's not worth it to move fetch in separate context (except you want to use single context for billing self-service connection state in multiple places)

Reply:
I removed this hook. Use react context with the loading of acquiring context once.